### ISSUE TITLE
BUG: endpoints aren't returning content-type=application/json

### ISSUE THREAD

**Issue by Slugger** [SchedulesDirect/JSON-Service#49]
The content-type for the /programs endpoint is set to application/json even though the response isn't valid json (each line of the response is json but the response as a whole is not valid json).  The content type should probably be set to text/plain.

**Comment by rkulagowski** [comment]
API 20141201 should now return proper JSON in all responses. Please confirm.

**Comment by Slugger** [comment]
So to validate, I disabled the "ignore header flag" I added to my client to work around this bug in the past.  The `/schedules` endpoint immediately fails.  It's returning `content-type: text/plain` when it shouldn't so I don't process the schedules, so I never get to asking for `/programs`.

I've updated the title of this bug.  Basically all endpoints are now supposed to return `content-type: application/json` and as of this moment, `/schedules` does not (but `/schedules/md5` does).
